crates/std_detect/src/detect/os/x86.rs

```diff
@@ -234,7 +234,7 @@ fn detect_features() -> cache::Initializer {
         // The `is_x86_feature_detected!("lzcnt")` macro then
         // internally maps to Feature::abm.
         enable(extended_proc_info_ecx, 5, Feature::abm);
-        if vendor_id == *b"AuthenticAMD" {
+        if vendor_id == *b"AuthenticAMD" || vendor_id == *b"HygonGenuine" {
             // These features are only available on AMD CPUs:
```

Comment:
This comment should be updated. It would be nice to add a link to AMD and Hygon cpuid specifications, so that it is easy to verify that these are the correct cpuid bits to read for these features in these CPUs.

Reply:
@hygonsoc did you forgot to push a comment resolving this comment ?

Reply:
@gnzlbg already push the commit resolving this comment, thanks!

Reply:
Hmm, I don't think its pushed. Like, I meant that the comment in the code should be updated with the information, not only the commit message 😆

Reply:
@gnzlbg sorry for the misunderstanding, update the comment in codes in commit f7da713, thanks!